### ISSUE TITLE
No tray icon on GNOME Shell 3.20.2

### ISSUE THREAD

**Issue by chmercesmoreira** [Aluxian/Whatsie#341]
* Operating system: Manjaro Gnome 16.06
* Whatsie version: 2.0.18

Hello, I'm installing the AUR version available here: https://aur.archlinux.org/packages/whatsie/
The application is installed correctly but there is no tray icon on GNOME shell, there are some users reporting this same issue on the comments section of the AUR package page. What could fix this issue, I have no idea, maybe this new version of Gnome Shell is not compatible.

**Comment by Aluxian** [comment]
The tray icon is disabled by default. Have you enabled it from Menu > Show in tray?

**Comment by chmercesmoreira** [comment]
This option is disabled by default so that I'm not able to enable it:
![Image]
(http://www.auplod.com/u/dlaopu82c4d.png)

**Comment by Aluxian** [comment]
Oh... the app thinks you're on Elementary OS. And I think I know why. Can you please run this in your terminal?

```
/bin/bash -c "cat /etc/os-release <(lsb_release -d) | grep \"elementary OS\""
```

**Comment by Aluxian** [comment]
Also, if you can tell, tell me that commands return code.

**Comment by chmercesmoreira** [comment]
The command returned no output.
`$ /bin/bash -c "cat /etc/os-release <(lsb_release -d) | grep \"elementary OS\""`
`$`
The content of **os-release**:
`$ cat /etc/os-release`
`NAME="Manjaro Linux"`
`ID=manjaro`
`PRETTY_NAME="Manjaro Linux"`
`ANSI_COLOR="1;32"`
`HOME_URL="http://www.manjaro.org/"`
`SUPPORT_URL="http://www.manjaro.org/"`
`BUG_REPORT_URL="http://bugs.manjaro.org/"`

**Comment by chmercesmoreira** [comment]
The command return code:
`$ /bin/bash -c "cat /etc/os-release <(lsb_release -d) | grep \"elementary OS\""`
`$ echo $?`
`1`

**Comment by Aluxian** [comment]
So probably running `lsb_release -d` fails on your OS? Aha! This must be it. I'll fix it. Damn, how did I not think about this...

**Comment by chmercesmoreira** [comment]
It doesn't fail:
```
$ lsb_release -d
Description:	Manjaro Linux
$ echo $?
0
```

**Comment by chmercesmoreira** [comment]
Nor the concatenation of the commands:
```
$ bash -c "cat /etc/os-release <(lsb_release -d)"
NAME="Manjaro Linux"
ID=manjaro
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="1;32"
HOME_URL="http://www.manjaro.org/"
SUPPORT_URL="http://www.manjaro.org/"
BUG_REPORT_URL="http://bugs.manjaro.org/"
Description:	Manjaro Linux
$ echo $?
0
```
What fails is the search for the string Elementary OS in the output of the previous command.

**Comment by Aluxian** [comment]
Ah, I missed something else. Fixed! Will be live in the next update.

**Comment by chmercesmoreira** [comment]
Ok, you can close this issue now. Thank you for answer me.

**Comment by Aluxian** [comment]
Fixed in 2.0.19-beta.
